src/pages/GetStarted/CustomWorkspaceTab/DevfileSelector/__tests__/index.spec.tsx

```diff
@@ -11,107 +11,254 @@
  */
 
 import { Provider } from 'react-redux';
-import { Store } from 'redux';
-import createMockStore from 'redux-mock-store';
+import { Action, Store } from 'redux';
 import mockAxios from 'axios';
 import React from 'react';
-import thunk from 'redux-thunk';
-import { RenderResult, render, screen, fireEvent } from '@testing-library/react';
+import { RenderResult, render, screen, fireEvent, waitFor } from '@testing-library/react';
 import DevfileSelectorFormGroup from '../';
-import { AppState } from '../../../../../store';
 import mockMetadata from '../../../__tests__/devfileMetadata.json';
+import { FakeStoreBuilder } from '../../../../../store/__mocks__/storeBuilder';
+import * as FactoryResolverStore from '../../../../../store/FactoryResolver';
+import { AppThunk } from '../../../../../store';
+
+jest.mock('../../../../../store/FactoryResolver.ts', () => {
+  return {
+    __esModule: true,
```

Comment:
Do we really need it in the mock?
```__esModule: true,```

Reply:
```
jest.mock('../../../../../store/FactoryResolver.ts', () => ({
    actionCreators: {
      requestFactoryResolver: (location: string): AppThunk<Action, Promise<void>> => async (): Promise<void> => {
        if (/nonexisting/.test(location)) {
          return Promise.reject();
        }
        return Promise.resolve();
      },
    } as FactoryResolverStore.ActionCreators,
  }
));

Reply:
it's not necessary here, I'm removing the field